lang/es.json

```diff
@@ -277,7 +279,7 @@
 	"label.done": "Listo",
 	"label.donor": "Donante",
 	"label.donors_to_higher_ranked_projects": "Los donantes de los proyectos con ranking más alto reciben más GIVbacks.",
-	"label.donors_to_verified": "Los donantes a proyectos verificados son recompensados con GIV. Boostea este proyecto para aumentar su porcentaje de recompensas y hacerlo más visible en la página de proyectos!",
+	"label.donors_to_verified": "Los donaciones en Ethereum a proyectos verificados son recompensados con GIV. Boostea este proyecto para aumentar su porcentaje de recompensas y hacerlo más visible en la página de proyectos!",
```

Comment:
Let's change it to

"Las donaciones en Ethereum a proyectos verificados son recompensadas con GIV. Boostea este proyecto para aumentar su porcentaje de recompensas y hacerlo más visible en la página de proyectos!"

Reply:
Thanks Mateo. I fixed it here https://github.com/Giveth/giveth-dapps-v2/pull/3689

Reply:
@mateodaza next time try amazing "Add suggestion" feature in your review, you would like it
<img width="258" alt="image" src="https://github.com/Giveth/giveth-dapps-v2/assets/5684607/b829acfc-4159-4000-b4db-1c48d715468d">
Then, the peer will push/apply your suggestion easily with a single button click.